scaffolding-chef-infra/tests/user-windows-api/habitat/plan.ps1

```diff
@@ -0,0 +1,18 @@
+#######################################
+# User plan for ci testing
+# This tests a user interacting with the API provided by the scaffolding
+#######################################
+
+$pkg_name="user-windows-api"
+$pkg_origin="ci"
+$pkg_version="1.0.0"
+$pkg_scaffolding="ci/scaffolding-chef-infra"
+$scaffold_policy_name="ci"
+$scaffold_cacerts="core/cacerts"
+$scaffold_chef_client="stuartpreston/chef-client-detox"
```

Comment:
oh I see. You are setting this var from the plan. In that case, the plan must also specify `$Script:scaffold_chef_client`, etc. All the "standard" plan variables are initialized with `SCRIPT` scope in `plan-build-ps1.ps1` but because these are "custom" variables, plan authors will need to do that. That feels kind of weird to me.

Actually what you could do in scaffolding.ps1 when you are inspecting these variables is copy them to script scoped variables so that they are accessible to the `Load-Scaffolding` function.